guides/common/modules/con_overview-of-hosts-in-project.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Overview-of-Hosts-in-{Project}_{context}"]
+= Overview of Hosts in {Project}
+
+A host is any Linux client that {ProjectName} manages.
+Hosts can be physical or virtual.
+
+ifndef::satellite,orcharhino[]
+You can deploy virtual hosts on any platform supported by {ProjectName}, such as Amazon EC2, Google Compute Engine, KVM, libvirt, Microsoft Azure, OpenStack, {oVirt}, Proxmox, Rackspace Cloud Services, or VMware vSphere.
+endif::[]
+ifdef::satellite[]
+You can deploy virtual hosts on any platform supported by {ProjectName}, such as Amazon EC2, Google Compute Engine, KVM, libvirt, Microsoft Azure, OpenStack, {oVirt}, Rackspace Cloud Services, or VMware vSphere.
+endif::[]
+ifdef::orcharhino[]
+You can deploy virtual hosts on any platform supported by {ProjectName}, such as Amazon EC2, Google Compute Engine, libvirt, Microsoft Azure, Oracle Linux Virtualization Manager, oVirt, Proxmox, RHV, and VMware vSphere.
+endif::[]
+
+With {Project}, you can manage hosts at scale, including monitoring, provisioning, remote execution, configuration management, software management, and subscription management.
+
+You can manage your hosts from the {ProjectWebUI} or from the command line.
```

Comment:
also via API and FAM

Reply:
Let's just delete line 19. The module is included in Planning_for_Project and Managing_Hosts and management tools are covered elsewhere in those guides.